draft/spec/index.html

```diff
@@ -459,6 +459,7 @@ <h2>Version Directories</h2>
             sub-directory. Once created, the contents of a version directory are expected to be immutable.
         </p>
         <section id="content-directory">
+          <h3>Content Directory</h3>
```

Comment:
I think this should be `<h2>` ... respec uses this same level for all headings that show in the TOC